src/net/http.rs

```diff
@@ -1568,7 +1568,10 @@ impl HttpRequestType {
                     } else {
                         "".to_string()
                     };
-                    info!("Handle {} {}{}", verb, decoded_path, query);
+                    info!(
+                        "Handle [{}] {} {}{}",
+                        verb, &protocol.peer_addr, decoded_path, query
+                    );
```

Comment:
Can this use the k-v logging syntax? i.e.,

```rust
info!("Handle HTTPRequest"; "verb" => %verb, "peer_addr" => %protocol.peer_addr, "path" => %decoded_path, "query" => %query);
```

Reply:
Done